src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -861,7 +861,7 @@ private void fillCitizenInfo(final Button button)
         final CitizenDataView view = citizens.get(row);
         WindowCitizen.createXpBar(view, this);
         WindowCitizen.createSkillContent(view, this);
-        findPaneOfTypeByID(JOB_LABEL, Label.class).setLabelText("§l" + LanguageHandler.format(view.getJob()));
+        findPaneOfTypeByID(JOB_LABEL, Label.class).setLabelText("§l" + LanguageHandler.format(view.getJob().trim().isEmpty() ? GUI_TOWNHALL_CITIZEN_JOB_UNEMPLOYED : view.getJob()));
```

Comment:
You sure getJob might not be NULL for unemployed sometimes?

Reply:
I tested it with logger on a colony with townhall and miner only. It worked
job = ByteBufUtils.readUTF8String(buf); <- I really don't know if this can return null or not
https://github.com/ldtteam/minecolonies/blob/cc3e597627862589c500e631b08024eab864c441/src/main/java/com/minecolonies/coremod/colony/CitizenDataView.java#L299

Reply:
I really don't want to:
since it makes if/else structure
it's not a solution to make an if(something is null) if it's not intended to work like that, it should simply return an empty string -> so I can through and make it working like that

Reply:
and for me, it works like that now